discovery_engine_core/web-api/src/models.rs

```diff
@@ -79,27 +83,30 @@ impl From<Document> for Article {
 /// Represents user interaction request body.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub(crate) struct InteractionRequestBody {
-    pub(crate) document_id: Id,
+    pub(crate) document_id: String,
```

Comment:
❔ Maybe we should also validating the `document_id` in the `handle_user_interaction` function. wdyt?

Reply:
it is used to lookup a value from `documents_by_id` HashMap, so it will return 404 if it won't find a value under provided key. Since we are validating article id when constructing this HashMap I would assume we don't have to add additional validation, but maybe I'm missing something?

Reply:
Makes sense 👍 then we don't need to check it